hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -2916,3 +2929,16 @@ func deleteConfigByLabel(ctx context.Context, c client.Client, lbl map[string]st
 	}
 	return nil
 }
+
+// validateHCPayloadSupportsNodePoolCPUArch validates the HostedCluster payload can support the NodePool's CPU arch
+func validateHCPayloadSupportsNodePoolCPUArch(hc *hyperv1.HostedCluster, np *hyperv1.NodePool) error {
+	if strings.EqualFold(hc.Status.Payload, "multi") {
```

Comment:
maybe we should have a const for "multi"

Reply:
I don't think it matters in this case. The payload is `multi`. Joel said the enum in the API should be capitalized so that introduces a discrepancy between those two things.

Reply:
then the const could be "Multi".
just a nit, no need to change it.